src/Domain/CreateInitialMesh.hpp

```diff
@@ -7,15 +7,18 @@
 #include <cstddef>
 #include <vector>
 
-#include "DataStructures/Index.hpp"
-#include "Domain/ElementId.hpp"
 #include "Domain/Mesh.hpp"
-#include "Domain/OrientationMap.hpp"
-#include "NumericalAlgorithms/Spectral/Spectral.hpp"
-#include "Utilities/Gsl.hpp"
 
-/// Items for initializing the DataBox%es of parallel components
+/// \cond
+template <size_t Dim>
+struct ElementId;
+template <size_t Dim>
+struct OrientationMap;
+/// \endcond
+
+namespace domain {
 namespace Initialization {
+/// \ingroup ComputationalDomainGroup
```

Comment:
I don't think you can put things in two doxygen groups; I would rather it show up in the initialization group

Reply:
You can with some things, but I don't remember which...

Reply:
Looks like it works for classes:

> compound entities (like classes, files and namespaces) can be put into multiple groups, but members (like variable, functions, typedefs and enums) can only be a member of one group (this restriction is in place to avoid ambiguous linking targets in case a member is not documented in the context of its class, namespace or file, but only visible as part of a group).

http://www.doxygen.nl/manual/grouping.html

So keep both groups?

Reply:
Travis is failing the documentation build with the statement that it will only use one group...

Reply:
Well in the documentation @nilsleiffischer copied it says functions can't be in two...

Reply:
Oh right I confused what this documentation is for. Well, only the init group then :)